.github/CODEOWNERS

```
@@ -1,41 +1,41 @@
 # Global Owners
-* @nickboldt @sparkoo
+* @nickboldt @ibuziuk @l0rd @sparkoo
 
 # dockerfiles
 dockerfiles/** @nickboldt
 
 # assembly
-assembly/** @skabashnyuk @nickboldt
-assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/** @ibuziuk @nickboldt @themr0c @boczkowska @MichalMaler
+assembly/** @nickboldt
+assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/** @ibuziuk @nickboldt @themr0c
 
 # core
-core/** @skabashnyuk @nickboldt
+core/** @nickboldt
 
 # workspace runtime infrastructure implementations
-infrastructures/** @skabashnyuk @amisevsk @metlos @nickboldt
+infrastructures/** @amisevsk @nickboldt
 
 # multiuser che
-multiuser/api/**  @skabashnyuk @nickboldt
-multiuser/integration-tests/**  @mshaposhnik @skabashnyuk @nickboldt
+multiuser/api/**  @nickboldt
+multiuser/integration-tests/**  @mshaposhnik @nickboldt
 multiuser/keycloak/**  @mshaposhnik @nickboldt
 multiuser/machine-auth/**  @mshaposhnik @nickboldt
-multiuser/permission/**  @skabashnyuk @nickboldt
-multiuser/personal-account/**  @skabashnyuk @nickboldt
+multiuser/permission/**  @nickboldt
+multiuser/personal-account/**  @nickboldt
 multiuser/sql-schema/**  @mshaposhnik @nickboldt
 
 # wsmaster
-wsmaster/** @skabashnyuk @nickboldt
+wsmaster/** @nickboldt
```

I would add @vinokurig here since he has been the main contributor during the last year or so:

```suggestion
wsmaster/** @nickboldt @vinokurig 
```